azurerm/internal/services/network/public_ip_resource_test.go

```diff
@@ -760,3 +776,28 @@ resource "azurerm_public_ip" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func (PublicIPResource) standard_IpTags(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
Recently, we've been adding the service package name into tests to better track where failing tests are coming from in the portal.

```suggestion
  name     = "acctestRG-network-%d"
```